### PR TITLE
Add Metroid II: Return of Samus

### PR DESCRIPTION
**Pull request by SuperNascher**
Adds Metroid II.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/metroid-ii/game.dat [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title = "Metroid II"
```
Comment (Leo626) [inline games/metroid-ii/game.dat]: This should be `Metroid II: Return of Samus`

**Comment by Leo626** [comment]
nit: prefer the directory to be called `metroid-ii-return-of samus`

**Comment by SuperNascher** [comment]
@Leo626 Could look at my commit again? I've adjusted the images, changed the names and added the wiki entry :-)

File: games/metroid-ii-return-of-samus/game.dat [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+release_date = "2011-11-24"
+
+[[testcases]]
+title = "0004000000050500"
+compatibility = "2"
+date = "2017-10-03"
+version = "HEAD-30fabc4"
+cpu = "Intel Core i5-4590"
+gpu = "Nvidia GeForce GTX 1080"
+os = "Arch Linux"
```
Comment (Leo626) [inline games/metroid-ii-return-of-samus/game.dat]: I'm not familiar with Linux and distributions but Arch Linux doesn't follow standard releases right? It doesn't have versions?

Comment (SuperNascher) [reply games/metroid-ii-return-of-samus/game.dat]: Yes, it has no versions because it's a rolling release distro.

Comment (chris062689) [reply games/metroid-ii-return-of-samus/game.dat]: Correct. Arch Linux is fine.

**Pull request merged**